pkg/sfu/buffer/buffer.go

```diff
@@ -159,6 +149,7 @@ func (b *Buffer) Bind(params webrtc.RTPParameters, codec webrtc.RTPCodecCapabili
 			switch fb.Type {
 			case webrtc.TypeRTCPFBGoogREMB:
 				b.logger.Debugw("Setting feedback", "type", webrtc.TypeRTCPFBGoogREMB)
+				b.logger.Warnw("REMB not supported, RTCP feedback will not be generated", nil)
```

Comment:
Warning about REMB here (it will print every time, but that is okay I think). The REMB building here is unused. It was incorrect anyway because REMB should be aggregated at peer connection level.